CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## NEXT VERSION
 
 * devel: add tini (init system)
+* devel: add net-tools(tools for controlling the network)
```

Comment:
- IMHO no need to describe  what `net-tools` package does
- not sure but maybe you want to add it also to `gh-runner` image?